Source/Common/compression.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
Is this used for anything besides compressing HTTP request bodies? would it make sense to move if from Source/Common to Source/HTTP?

Reply:
We can totally move it if it makes more sense! Working on it

Reply:
Already moved both files.